sdks/python/apache_beam/metrics/metric.py

```diff
@@ -139,6 +169,22 @@ def __init__(self, metric_name):
       super(Metrics.DelegatingGauge, self).__init__(metric_name)
       self.set = MetricUpdater(cells.GaugeCell, metric_name)  # type: ignore[assignment]
 
+  class DelegatingHistogram(Histogram):
```

Comment:
Please move histogram related code into a new 'internal' file
sdks/python/apache_beam/internal/metrics/metric.py

Please move DelegatingHistogram to the new file

add a Metric class to the new file, and put the histogram definition there.

It can share the infrastructure and call things in this file.

Reply:
done.